app/Services/AuthService.php

```
@@ -0,0 +1,223 @@
+        if ($oldJwt && JWTAuth::setToken($oldJwt)->check()) {
+            JWTAuth::setToken($oldJwt)->invalidate();
+            Cache::forget($jwtKey);
+        }
+    }
+
+    private function generateResetPasswordUrl($link)
+    {
+        // Implement the logic to generate the full reset password URL
+        // This is a placeholder and should be replaced with the actual implementation
+        return $link;
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'generateResetPasswordUrl'. (Unused Code Rules)

(UnusedPrivateMethod)